22:46:13,160 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:13,196 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:13,206 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:13,215 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-35 to job manager
22:46:13,280 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:13,280 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:13,281 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:14,392 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:14,411 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:14,420 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:14,420 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,425 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (20/25)
22:46:46,427 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (20/25)
22:46:49,345 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (158/200)
22:46:49,345 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (158/200)
22:46:49,347 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (158/200)
22:46:49,348 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (158/200)
22:46:49,901 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (156/200)
22:46:49,902 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (156/200)
22:46:49,903 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (156/200)
22:46:49,904 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (156/200)
22:46:49,948 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (155/200)
22:46:49,949 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (155/200)
22:46:49,951 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (155/200)
22:46:49,953 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (155/200)
22:46:49,995 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (153/200)
22:46:49,996 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (153/200)
22:46:49,998 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (153/200)
22:46:49,999 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (153/200)
22:46:50,715 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (157/200)
22:46:50,715 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (157/200)
22:46:50,717 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (157/200)
22:46:50,718 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (157/200)
22:46:51,143 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (159/200)
22:46:51,143 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (159/200)
22:46:51,145 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (159/200)
22:46:51,146 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (159/200)
22:46:51,457 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (160/200)
22:46:51,457 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (160/200)
22:46:51,459 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (160/200)
22:46:51,460 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (160/200)
22:46:52,489 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (154/200)
22:46:52,489 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (154/200)
22:46:52,491 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (154/200)
22:46:52,493 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (154/200)
22:47:20,404 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (20/25)
22:47:20,405 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (20/25)
22:49:30,916 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2213111204361479974_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:39998 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:30,917 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2213111204361479974_1722 bad datanode[0] 130.149.21.65:45020
22:49:30,918 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2213111204361479974_1722 in pipeline 130.149.21.65:45020, 130.149.21.30:45020, 130.149.21.23:45020: bad datanode 130.149.21.65:45020
22:49:32,631 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_6415024047390416788_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:40003 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:32,632 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6415024047390416788_1722 bad datanode[0] 130.149.21.65:45020
22:49:32,632 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6415024047390416788_1722 in pipeline 130.149.21.65:45020, 130.149.21.28:45020, 130.149.21.23:45020: bad datanode 130.149.21.65:45020
22:49:33,964 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-820948111546090973_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:40011 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:33,965 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-820948111546090973_1722 bad datanode[0] 130.149.21.65:45020
22:49:33,966 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-820948111546090973_1722 in pipeline 130.149.21.65:45020, 130.149.21.28:45020, 130.149.21.27:45020: bad datanode 130.149.21.65:45020
22:49:50,175 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-870454692905249355_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:40001 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:50,176 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-870454692905249355_1722 bad datanode[0] 130.149.21.65:45020
22:49:50,176 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-870454692905249355_1722 in pipeline 130.149.21.65:45020, 130.149.21.17:45020, 130.149.21.19:45020: bad datanode 130.149.21.65:45020
22:49:51,317 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_3205533779724946536_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:39995 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:51,318 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3205533779724946536_1722 bad datanode[0] 130.149.21.65:45020
22:49:51,319 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3205533779724946536_1722 in pipeline 130.149.21.65:45020, 130.149.21.24:45020, 130.149.21.20:45020: bad datanode 130.149.21.65:45020
22:50:01,052 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1672519878979910803_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:39993 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:01,053 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1672519878979910803_1722 bad datanode[0] 130.149.21.65:45020
22:50:01,054 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1672519878979910803_1722 in pipeline 130.149.21.65:45020, 130.149.21.64:45020, 130.149.21.17:45020: bad datanode 130.149.21.65:45020
22:50:03,050 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1316005299982734336_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:40007 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:03,051 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1316005299982734336_1722 bad datanode[0] 130.149.21.65:45020
22:50:03,052 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1316005299982734336_1722 in pipeline 130.149.21.65:45020, 130.149.21.20:45020, 130.149.21.61:45020: bad datanode 130.149.21.65:45020
22:50:04,380 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-5916226431285061136_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:40009 remote=/130.149.21.65:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:04,381 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5916226431285061136_1722 bad datanode[0] 130.149.21.65:45020
22:50:04,382 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5916226431285061136_1722 in pipeline 130.149.21.65:45020, 130.149.21.27:45020, 130.149.21.66:45020: bad datanode 130.149.21.65:45020
22:50:36,999 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2213111204361479974_1772java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:59275 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:37,000 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2213111204361479974_1772 bad datanode[0] 130.149.21.30:45020
22:50:37,000 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2213111204361479974_1772 in pipeline 130.149.21.30:45020, 130.149.21.23:45020: bad datanode 130.149.21.30:45020
22:50:38,730 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_6415024047390416788_1778java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:33498 remote=/130.149.21.28:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:38,731 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6415024047390416788_1778 bad datanode[0] 130.149.21.28:45020
22:50:38,731 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6415024047390416788_1778 in pipeline 130.149.21.28:45020, 130.149.21.23:45020: bad datanode 130.149.21.28:45020
22:50:40,066 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-820948111546090973_1780java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:33500 remote=/130.149.21.28:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:40,068 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-820948111546090973_1780 bad datanode[0] 130.149.21.28:45020
22:50:40,068 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-820948111546090973_1780 in pipeline 130.149.21.28:45020, 130.149.21.27:45020: bad datanode 130.149.21.28:45020
22:50:56,274 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-870454692905249355_1801java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:58648 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:56,275 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-870454692905249355_1801 bad datanode[0] 130.149.21.17:45020
22:50:56,276 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-870454692905249355_1801 in pipeline 130.149.21.17:45020, 130.149.21.19:45020: bad datanode 130.149.21.17:45020
22:50:57,418 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_3205533779724946536_1806java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:60014 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:57,419 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3205533779724946536_1806 bad datanode[0] 130.149.21.24:45020
22:50:57,419 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3205533779724946536_1806 in pipeline 130.149.21.24:45020, 130.149.21.20:45020: bad datanode 130.149.21.24:45020
22:51:07,154 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1672519878979910803_1823java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:56377 remote=/130.149.21.64:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:07,156 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1672519878979910803_1823 bad datanode[0] 130.149.21.64:45020
22:51:07,156 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1672519878979910803_1823 in pipeline 130.149.21.64:45020, 130.149.21.17:45020: bad datanode 130.149.21.64:45020
22:51:09,152 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1316005299982734336_1825java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:45983 remote=/130.149.21.20:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:09,153 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1316005299982734336_1825 bad datanode[0] 130.149.21.20:45020
22:51:09,154 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1316005299982734336_1825 in pipeline 130.149.21.20:45020, 130.149.21.61:45020: bad datanode 130.149.21.20:45020
22:51:10,498 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-5916226431285061136_1830java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.65:35246 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:10,499 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5916226431285061136_1830 bad datanode[0] 130.149.21.27:45020
22:51:10,500 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5916226431285061136_1830 in pipeline 130.149.21.27:45020, 130.149.21.66:45020: bad datanode 130.149.21.27:45020
